pkg/antctl/raw/supportbundle/command.go

```diff
@@ -104,7 +107,8 @@ func init() {
 		Command.Flags().StringVarP(&option.dir, "dir", "d", "", "support bundles output dir, the path will be created if it doesn't exist")
 		Command.Flags().StringVarP(&option.labelSelector, "label-selector", "l", "", "selector (label query) to filter Nodes for agent bundles, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 		Command.Flags().BoolVar(&option.controllerOnly, "controller-only", false, "only collect the support bundle of Antrea controller")
-		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of Antrea controller")
+		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of specific nodes filtered by names in a file (one node name per line)")
+		Command.Flags().StringVarP(&option.since, "since", "","", "only return logs newer than a relative duration like 5s, 2m, 3h, 4d, 1w(weeks). Defaults to all logs")
```

Comment:
Just a question here,  what does 1w(weeks) mean? Does it mean we need to use weeks if it is more than 1?  I guess not since others like second are not following this convention.

Reply:
No. It means we can use weeks as a unit for our duration. Like 1s, 1d, 1m, 1w, alough it's not very common. The go library i used support it, so I think it would be nice to expose it to the use.

Reply:
Got it, it is just for explaining what w means. Thanks for the explanation.

Reply:
I wonder if the user will get confused too...

Reply:
Personally, I think w itself is good enough for understanding. Another question, month and year are not supported? 2m here can only mean 2 minutes right?

Reply:
The should follow the golang time.Duration convention. time.Duration does not have const for week or month. So I think it should be ok (for users known about golang)

Reply:
`d` is userful, may be we can remove the `w`

Reply:
Swiched to standard time package, `d` and `w` is no longer supported